views/account/profile/index.ejs

```diff
@@ -14,33 +25,32 @@
     >
       <div class="row col-12">
         <main class="px-3">
-          <h1 class="display-4">Goon up Brah.</h1>
-          <h1>Hello World!</h1>
-          <p>This file was served with Express.</p>
-          <p>Here is your information:</p>
-          <ul>
-            <li>First Name: <%= user.first_name %></li>
-            <li>Last Name: <%= user.last_name %></li>
-            <li>Email: <%= user.email %></li>
-            <li>Password: <%= user.password %></li>
-            <li>
-              Settings:
-              <ul>
-                <% var settings = JSON.parse(user.settings);
-                Object.keys(settings).forEach( (key) => { %>
-                <li><%= key %>: <%= settings[key] %></li>
-                <% }); %>
-              </ul>
-            </li>
-          </ul>
-
-          <a
-            href="#"
-            class="btn btn-warning btn-lg"
-            role="button"
-            aria-disabled="true"
-            >Generate</a
-          >
+          <h1 class="display-4">Profile</h1>
+          <h1 id="casual-header">Casual</h1>
```

Comment:
I also checked the tables on mobile, since that's where a majority of this stuff is going to happen. The second table works well and has buttons for expanding and reducing entries, but the first table is a little wonky. If there is a setting to make it more mobile-friendly like the first table, that would help make things look a little better!

Here is what the first table looks like:

![IMG_2780](https://user-images.githubusercontent.com/57500301/198897940-0e46c6d5-94c3-4df2-8517-5a7cf83fadab.PNG)


Here is what the second table looks like:

![IMG_2781](https://user-images.githubusercontent.com/57500301/198897912-df9e58fd-18ab-4301-a259-a3404a153ee3.PNG)